README.md

```diff
@@ -6,49 +6,47 @@
+- **[Event handler: API Gateway and ALB](https://awslabs.github.io/aws-lambda-powertools-python/latest/core/event_handler/api_gateway/)** - Amazon API Gateway REST/HTTP API and ALB event handler for Lambda functions invoked using Proxy integration
+- **[Bring your own middleware](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/middleware_factory/)** - Decorator factory to create your own middleware to run logic before, and after each Lambda invocation
+- **[Parameters utility](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/parameters/)** - Retrieve and cache parameter values from Parameter Store, Secrets Manager, or DynamoDB
+- **[Batch processing](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/batch/)** - Handle partial failures for AWS SQS batch processing
+- **[Typing](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/typing/)** - Static typing classes to speedup development in your IDE
+- **[Validation](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/validation/)** - JSON Schema validator for inbound events and responses
+- **[Event source data classes](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/data_classes/)** - Data classes describing the schema of common Lambda event triggers
+- **[Parser](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/parser/)** - Data parsing and deep validation using Pydantic
+- **[Idempotency](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/idempotency/)** - Convert your Lambda functions into idempotent operations which are safe to retry
+- **[Feature Flags](https://awslabs.github.io/aws-lambda-powertools-python/latest/utilities/feature_flags/)** - A simple rule engine to evaluate when one or multiple features should be enabled depending on the input
 
 ### Installation
 
-With [pip](https://pip.pypa.io/en/latest/index.html) installed, run: ``pip install aws-lambda-powertools``
+With [pip](https://pip.pypa.io/en/latest/index.html) installed, run: `pip install aws-lambda-powertools`
 
 ## Examples
 
-* [Serverless Shopping cart](https://github.com/aws-samples/aws-serverless-shopping-cart)
-* [Serverless Airline](https://github.com/aws-samples/aws-serverless-airline-booking)
-* [Serverless E-commerce platform](https://github.com/aws-samples/aws-serverless-ecommerce-platform)
+- [Serverless Shopping cart](https://github.com/aws-samples/aws-serverless-shopping-cart)
+- [Serverless Airline](https://github.com/aws-samples/aws-serverless-airline-booking)
+- [Serverless E-commerce platform](https://github.com/aws-samples/aws-serverless-ecommerce-platform)
+- [Serverless GraphQL Nanny Booking Api](https://github.com/trey-rosius/babysitter_api)
```

Comment:
keep this change, revert all else ;)

Reply:
Hi @heitorlessa I apologize, I should've checked the file changes properly.  
I've reverted and made the correct change. 
Please let me know if I'm on track. 🙏🏿